lib/components/app/responsive-webapp.js

```diff
@@ -209,12 +207,12 @@ class RouterWrapperWithAuth0 extends Component {
       showAccessTokenError,
       showLoginError
     } = this.props
-
+    console.log(this.props)
     const router = (
       <ConnectedRouter
         basename={routerConfig && routerConfig.basename}
         history={history}>
-        <div>
+        <>
```

Comment:
Remove `<>` and make `<Switch>` the child element.

Reply:
Fixed in bbb6987f3a529c8851d6d7e868fabade4632872d.